apps/designer/app/designer/features/style-panel/controls/color/color-control.tsx

```diff
@@ -1,29 +1,59 @@
 import { Flex } from "@webstudio-is/design-system";
+import { toValue } from "@webstudio-is/css-engine";
 import { getFinalValue } from "../../shared/get-final-value";
 import type { ControlProps } from "../../style-sections";
 import { ColorPicker } from "../../shared/color-picker";
+import { colord } from "colord";
 
 export const ColorControl = ({
   currentStyle,
   inheritedStyle,
   setProperty,
   styleConfig,
 }: ControlProps) => {
-  const value = getFinalValue({
+  let value = getFinalValue({
     currentStyle,
     inheritedStyle,
     property: styleConfig.property,
   });
+
   if (value === undefined) {
     return null;
   }
+
   const setValue = setProperty(styleConfig.property);
 
+  if (value.type !== "rgb") {
+    // Support previously set colors
+    const colordValue = colord(toValue(value));
+
+    if (colordValue.isValid()) {
+      const rgb = colordValue.toRgb();
+      value = {
+        type: "rgb",
+        r: rgb.r,
+        g: rgb.g,
+        b: rgb.b,
+        alpha: rgb.a ?? 100,
```

Comment:
alpha is either percentage or 0-1, if you don't use percentage then maybe we should use 1?